### DISCUSSION TITLE
plugin grpc-gateway can't import proto file

### DISCUSSION THREAD

**Discussion by YeRuyi** [Kong/kong#7227] [Help]
I used kong as docker container, and setting grpc-gateway plugin for a router. Plugin seted one proto file within import `google/protobuf/xxxx.proto` for resolving, but runtime can not find the file and import it.

How can i using it?

1. I try copy lib to `/usr/local/include`, but not fixed.
2. I try to copy lib file to runtime path. and not effected, too.

.proto file 

```
syntax = "proto3";

package svr.cal.v1;

import "google/api/annotations.proto";
import "google/protobuf/any.proto";

service CalPubSvr {

  rpc GetStableCal(GetStableCalReq)returns(GetStableCalRes){
    option (google.api.http) = {
      post: "/v1/getStableCal"
      body: "*"
    };
  }
}

....
```

**Comment by javierguerragiraldez** [comment]
grpc-gateway 1.2  adds the `import` feature.  it uses the directory holding your .proto file as import root

**Comment by YeRuyi** [comment]
Thx for your help. Btw, how can i use grpc-gateway plugin with version 1.2 as default? I using docker continer with kong:2.2.0-centos, and plugin be set version with 0.1.1-2 as default.

@javierguerragiraldez
